modules/ossm-about-adding-namespace.adoc

```diff
@@ -0,0 +1,41 @@
+
+[NOTE]
+====
+In OpenShift Container Platform, a project is essentially a Kubernetes namespace with additional annotations, such as the range of user IDs that can be used in the project.
+====
+
+You can add projects to an existing service mesh using either the web console or the CLI. There are two methods to add a project to a mesh:
+
+* Specifying a project in the `ServiceMeshMemberRoll` resource.
+
+* Creating a `ServiceMeshMember` resource in a project.
+
+.ServiceMeshMemberRoll method
+
+This is the simplest way to add a project to a service mesh. To add a project, specify the project namespace where your applications reside in the `spec.members` field of the `ServiceMeshMemberRoll` resource.
+
+[NOTE]
+====
+Adding projects using this method requires a user to have mesh admin privileges.
+====
+
+The items listed in the `ServiceMeshMemberRoll` resource are the applications and workflows that are managed by the `ServiceMeshControlPlane` resource. The control plane, which includes the {SMProductShortName} Operators, Istiod, and `ServiceMeshControlPlane`, and the data plane, which includes applications and the Envoy proxy, must be in separate namespaces.
+
+* If you already have an application, workload, or service to add to a `ServiceMeshMemberRoll` resource, see the instructions for adding or removing projects from the service mesh using the `ServiceMeshMemberRoll` resource with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-add-project-member-roll-recourse-console_ossm-create-mesh[web console] or with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-add-project-member-roll-resource-cli_ossm-create-mesh[CLI]. 
+
+* Alternatively, to install a sample application called Bookinfo and add it to a `ServiceMeshMemberRoll` resource, skip to the tutorial for installing the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo example application] to see how an application works in {SMProductName}.
+
+.ServiceMeshMember method
+
+A `ServiceMeshMember` resource provides a way to add a project to a service mesh without modifying the `ServiceMeshMemberRoll` resource. To add a project, create a `ServiceMeshMember` resource in the project that you want to add to the service mesh. The namespace that contains the `ServiceMeshMember` resource becomes part of the `ServiceMeshMemberRoll` resource, and the services that reside in the namespace are made available to the mesh. For more information, see the instructions for adding projects to the service mesh using the `ServiceMeshMember` resource with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-adding-project-using-smm-resource-console_ossm-create-mesh[web console] or with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-adding-project-using-smm-resource-cli_ossm-create-mesh[CLI].
```

Comment:
I can't find the proper wording, but this needs to be changed so that it's clear that the user is creating the resource in the project that they are adding to the mesh. Something like:
```suggestion
A `ServiceMeshMember` resource provides a way to add a project to a service mesh without modifying the `ServiceMeshMemberRoll` resource. To add a project, create a `ServiceMeshMember` resource in that same project. When the `ServiceMeshMember` object is processed by the {SMProductShortName} Operator, the project appears in the `status.members` list in the `ServiceMeshMemberRoll` resource, and the services that reside in the project are made available to the mesh. For more information, see the instructions for adding projects to the service mesh using the `ServiceMeshMember` resource with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-adding-project-using-smm-resource-console_ossm-create-mesh[web console] or with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-adding-project-using-smm-resource-cli_ossm-create-mesh[CLI].
```